message/validation/partial_validation.go

```diff
@@ -64,6 +65,7 @@ func (mv *messageValidator) validatePartialSignatureMessage(
 
 	if msgSlot > signerState.Slot {
 		newEpoch := mv.netCfg.Beacon.EstimatedEpochAtSlot(msgSlot) > mv.netCfg.Beacon.EstimatedEpochAtSlot(signerState.Slot)
+		mv.logger.Info("Resetting slot", zap.Int("signer", int(signedMsg.Signer)), zap.String("validator", string(msgID.GetPubKey())), zap.Int("slot", int(msgSlot)), zap.Int("round", int(specqbft.FirstRound)), zap.Bool("new epoch", newEpoch))
```

Comment:
I guess you want to log the hex string of the validator as this looks bad..

Reply:
@MatheusFranco99

Reply:
Should be removed. It was just used for debugging